Gemfile

```diff
@@ -30,6 +30,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   # Make rspec output shorter and more useful
   gem 'fivemat'
+  gem 'memory_profiler'
```

Comment:
This isn't used during automated tests, so it just needs to be in the `development` group.